serverless/serverless
# Quickfix "too many open files" issue on Windows

## What did you implement:

Closes #6557

Refs #6636

## How did you implement it:

Globally enable `graceful-fs`. This PR is a quickfix to ensure that the bugfix is shipped in v1.52.0. However #6636 is a follow-up which consolidates the 2 different zipping functionalities we're currently using.

## How can we verify it:

Just look at the code and see that the tests are still green. You might want to package / deploy a service with a custom resource (e.g. `eventBridge` event) on Windows to ensure that this PR fixes the problem.

## Todos:

_**Note: Run `npm run test-ci` to run all validation checks on proposed changes**_

       **Validate via `npm test`**
       **Validate via `npm run lint-updated`**  
       _Note: Some reported issues can be automatically fixed by running `npm run lint:fix`_
       **Validate via `npm run prettier-check-updated`**  
       _Note: All reported issues can be automatically fixed by running `npm run prettify-updated`_

**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO
